lib/build-asset-map-reader.js

```diff
@@ -5,7 +5,7 @@ let fs = require("fs");
 // An asset map reader looks up assets in a manifest file
 module.exports = assetMaps => {
 	return () => {
-		return Promise.all(assetMaps.map(readJSON)
+		return Promise.all(assetMaps.map(retry(readJSON, [10, 100, 1000]))
```

Comment:
I'd prefer something like `[10, 50, 100, 250, 500, 100]`, as otherwise chances of unnecessary noticeable delays are fairly high.